Source/WebCore/css/SelectorCheckerTestFunctions.h

```diff
@@ -248,6 +248,27 @@ ALWAYS_INLINE bool matchesLangPseudoClass(const Element& element, const Vector<A
     return false;
 }
 
+#if ENABLE(CSS_SELECTORS_LEVEL4)
```

Comment:
Do you know why `CSS_SELECTORS_LEVEL4` is enabled only for Cocoa platforms? Seems strange to me that some core CSS functionality is not enabled everywhere.

Reply:
Tracking that in https://bugs.webkit.org/show_bug.cgi?id=243092